repository/expression.js

```diff
@@ -1,6 +1,24 @@
 import { Expression, User, UserExpression } from '../models';
 
 class ExpressionRepository {
+  getAllExpressions() {
+    return User.findAll({
```

Comment:
I don't quite understand what this is doing. I see that you are getting a list of users with their expressions, but why isn't it in the user repo since you are making a query on the users table. Either query expressions and include on each expressions the username of the user or move this in the user repo and maybe rename it `getUsersWithExpressions`